modules/multi-architecture-modify-machine-set-gcp.adoc

```diff
@@ -0,0 +1,150 @@
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/<role>: ""
+      providerSpec:
+        value:
+          apiVersion: gcpprovider.openshift.io/v1beta1
+          canIPForward: false
+          credentialsSecret:
+            name: gcp-cloud-credentials
+          deletionProtection: false
+          disks:
+          - autoDelete: true
+            boot: true
+            image: <path_to_image> <3>
+            labels: null
+            sizeGb: 128
+            type: pd-ssd
+          gcpMetadata: <4>
+          - key: <custom_metadata_key>
+            value: <custom_metadata_value>
+          kind: GCPMachineProviderSpec
+          machineType: n1-standard-4 <5>
+          metadata:
+            creationTimestamp: null
+          networkInterfaces:
+          - network: <infrastructure_id>-network
+            subnetwork: <infrastructure_id>-worker-subnet
+          projectID: <project_name> <6>
+          region: us-central1 <7>
+          serviceAccounts:
+          - email: <infrastructure_id>-w@<project_name>.iam.gserviceaccount.com
+            scopes:
+            - https://www.googleapis.com/auth/cloud-platform
+          tags:
+            - <infrastructure_id>-worker
+          userDataSecret:
+            name: worker-user-data
+          zone: us-central1-a
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+<2> Specify the role node label to add.
+<3> Specify the path to the image that is used in current compute machine sets. You need the project and image name for your path to image. 
++
+To access the project and image name, run the following command: 
++ 
+[source,terminal]
+----
+$ oc get configmap/coreos-bootimages \ 
+  -n openshift-machine-config-operator \ 
+  -o jsonpath='{.data.stream}' | jq \ 
+  -r '.architectures.aarch64.images.gcp'
+----
++
+.Example output 
+[source,terminal]
+----
+  "gcp": {
+    "release": "415.92.202309142014-0",
+    "project": "rhcos-cloud",
+    "name": "rhcos-415-92-202309142014-0-gcp-aarch64"
+  }
+----
+Use the `project` and `name` parameters from the output to create the path to image field in your machine set. The path to the image should follow the following format:
++
+[source,terminal]
+----
+$ projects/<project>/global/images/<image_name>
+----
+<4> Optional: Specify custom metadata in the form of a `key:value`` pair. For example use cases, see the GCP documentation for link:https://cloud.google.com/compute/docs/metadata/setting-custom-metadata[setting custom metadata].
+<5> Specify an ARM64 supported machine type. For more information, refer to "Tested instance types for GCP 64-bit ARM"
```

Comment:
```suggestion
<5> Specify an ARM64 supported machine type. For more information, refer to "Additional resources".
```
I'd maybe just point to the Additional resources section. Reading through this at first I thought, _where would I find those docs_? 🙂